package.json

```diff
@@ -19,7 +21,7 @@
     "react-datepicker": "4.13.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.12.1",
-    "sort-by": "^1.2.0"
+    "sort-by": "^0.0.2"
```

Comment:
Why downgrading this ? I've used it to integrate modern routing, and I think to remove it, is there any purpose to keep it ?